docs/index.html

```diff
@@ -5,14 +5,15 @@
         <meta name="viewport" content="width=device-width, initial-scale=1"/>
         <link rel="stylesheet" type="text/css" href="styles/style.css"/>
         <script src="script.js"></script>
-        <link href="index.html" >
+        <script src="my.js"></script>
+        
       
     </head>
     <body>
 
         <p class="dayclock" id="datetime"></p>
         <div class="headher">
-            <h1> Welcome to My Web Page!!</h1>
+            <h> Welcome to My Web Page!!</h>
```

Comment:
There is no `<h>` element in HTML.  `<h1>` was correct.  If you want to emphasize text without making it a header you should use `<strong>` or `<em>`